decorators/celery_.py

```diff
@@ -10,12 +11,24 @@
 from htk.utils.notifications import slack_notify
 from htk.utils.timer import HtkTimer
 
+# Zippy Imports
+from zippy.celeryconfig import app as celery_app
```

Comment:
Zippy is not an existing module inside `django-htk`.
`django-htk` is a general purpose Tool for Django apps. Whatever we change in htk, we make sure that it works for all projects without depending any Django projects that using `django-htk`

Reply:
I see, thank you. I will come up with a different way to handle this logic.